source/adapters/cuda/kernel.cpp

```diff
@@ -299,6 +299,7 @@ UR_APIEXPORT ur_result_t UR_APICALL urKernelGetInfo(ur_kernel_handle_t hKernel,
                                       hKernel->get()));
     return ReturnValue(static_cast<uint32_t>(NumRegs));
   }
+  case UR_KERNEL_INFO_SPILL_MEM_SIZE:
```

Comment:
This should return `UR_RESULT_ERROR_UNSUPPORTED_ENUMERATION` rather than `UR_RESULT_ERROR_INVALID_ENUMERATION`.

Reply:
done